src/components/Map/GenericMap.tsx

```diff
@@ -17,13 +20,20 @@ type GenericMapProps = {
   contextRenderer?: MapPopupRenderer;
   options?: MapOptions;
   style?: object;
+  organization?: ServerOrganization;
```

Comment:
Looks good.
can we just pass the message in here instead?
the context of showing this message is in PlantingSiteDashboardMap - which is where we should make the decision to set the message

Reply:
On second thought, leave it this way for now.